lib/std/mem.zig

```diff
@@ -8,11 +8,31 @@ const meta = std.meta;
 const trait = meta.trait;
 const testing = std.testing;
 
-pub const page_size = switch (builtin.arch) {
+pub const min_page_size = switch (builtin.arch) {
+    .mips, .mipsel, .mips64, .mips64el => switch (builtin.os) {
+        else => 1024, // NEC VR41xx processors support as low as 1K page size
+        .linux => 4 * 1024, // Linux doesn't support < 4K pages
+    },
+    .sparcv9 => 8 * 1024,
     .wasm32, .wasm64 => 64 * 1024,
     else => 4 * 1024,
 };
 
+pub const max_page_size = switch (builtin.arch) {
```

Comment:
can you justify the inclusion of this constant? The one place you used it, I disagree with. That test should be using actual page size.

Reply:
What do you mean by "actual page size"?